pkg/validation/internal/removed_apis.go

```diff
@@ -40,6 +40,11 @@ var K8sVersionsSupportedByValidator = []string{"1.22.0", "1.25.0", "1.26.0"}
 // - 1.25 : https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-25
 //
 // - 1.26 : https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-26
+//
+// IMPORTANT: Note that in the case scenarios of 1.25 and 1.26 it is very unlikely the OperatorAuthors
+// add manifests on the bundle using these APIs. On top of that some Kinds such as the CronJob
+// are not currently a valid/supported by OLM and never would to be added to bundle.
+// See: https://github.com/operator-framework/operator-registry/blob/v1.19.5/pkg/lib/bundle/supported_resources.go#L3-L23
```

Comment:
nit: Do we have any upstream docs on supported resources? It would be nice to have that to link to, instead of in the code somewhere. I couldn't find anything.